include/iocore/cache/CacheVC.h

```diff
@@ -141,6 +141,9 @@ struct CacheVC : public CacheVConnection {
 
   int handleReadDone(int event, Event *e);
   int handleRead(int event, Event *e);
+  bool check_ram_cache();
```

Comment:
Can we make this a const function?

Reply:
Great idea!

Reply:
However, they are not `const` methods, because they create or assign to the CacheVC buffer.

Reply:
The methods I added to `Stripe` and `AggregateWriteBuffer` should have been marked `const`, though. This has now been fixed.

Reply:
I missed that they're changing member variables. Let's leave it.

For the `Stripe` and `AggregateWriteBuffer`, marking `const` looks good.